chapters/equations.tex

```diff
@@ -181,11 +181,20 @@ \subsection{If-Equations}\label{if-equations}
 In an equation section, the equations in the body are seen as equations that must be satisfied.
 The bodies that are not selected have no effect on that model evaluation.
 
-The \lstinline!if!-equations in equation sections which do not have exclusively parameter expressions as switching conditions shall have the same number of equations in each branch (a missing else is counted as zero equations and the number of equations is defined as in \cref{local-equation-size}).
+The \lstinline!if!-equations in equation sections which do not have exclusively parameter expressions as switching conditions shall satisfy the following:
+\begin{itemize}
+\item
+  Have the same number of equations in each branch (a missing else is counted as zero equations and the number of equations is defined as in \cref{local-equation-size}).
+\item
+  If the equations are non-Real they shall have component-references (or a list of them) as their left-hand-side.
+  Any indices for such component-references must be evaluable.
```

Comment:
I think we call then _subscripts_, compare `\cref{connect-equations-and-connectors}`:

```suggestion
  Any subscripts for such component-references must be evaluable.
```